### ISSUE TITLE
Stardew Valley Update v.1.5.52

### ISSUE THREAD

**Issue by dorothydiaz** [ZaneYork/SMAPI-Android-Installer#614]
This version of SMAPI is not compatible with the latest version that I've installed from playstore.

**Comment by GonzaUwU** [comment]
> This version of SMAPI is not compatible with the latest version that I've installed from playstore.

If it is compatible, you just have to reinstall smapi and install it with "MTManager"

![Screenshot_20231109-223027](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/97956553/a72501bc-3821-4507-b500-83ae6444011d)

**Comment by 3Negativ3** [comment]
> > This version of SMAPI is not compatible with the latest version that I've installed from playstore.
> 
> If it is compatible, you just have to reinstall smapi and install it with "MTManager"
> 
> ![Screenshot_20231109-223027](https://user-images.githubusercontent.com/97956553/281926791-a72501bc-3821-4507-b500-83ae6444011d.png)

Thanks it works 😁

**Comment by dorothydiaz** [comment]
> > This version of SMAPI is not compatible with the latest version that I've installed from playstore.
> 
> If it is compatible, you just have to reinstall smapi and install it with "MTManager"
> 
> ![Screenshot_20231109-223027](https://user-images.githubusercontent.com/97956553/281926791-a72501bc-3821-4507-b500-83ae6444011d.png)

still not working 🥺

**Comment by LWGITHUBID** [comment]
> > > This version of SMAPI is not compatible with the latest version that I've installed from playstore.
> > 
> > 
> > If it is compatible, you just have to reinstall smapi and install it with "MTManager"
> > ![Screenshot_20231109-223027](https://user-images.githubusercontent.com/97956553/281926791-a72501bc-3821-4507-b500-83ae6444011d.png)
> 
> still not working 🥺

I think. We have to wait for the developers to have time to make it.

**Comment by GiLLo15** [comment]
Please and update 🙏😥

**Comment by Weaky322** [comment]
The latest Play Store version is not compatible, even though the SMAPI installer APK tells you to use 1.5.6.39 **or higher**.

**Comment by Rukkung1** [comment]
@ZaneYork Should you know this?

**Comment by Nhincaije** [comment]
> > This version of SMAPI is not compatible with the latest version that I've installed from playstore.
> 
> If it is compatible, you just have to reinstall smapi and install it with "MTManager"
> 
> ![Screenshot_20231109-223027](https://github.com/ZaneYork/SMAPI-Android-Installer/assets/97956553/a72501bc-3821-4507-b500-83ae6444011d)
> 

Can you be more specific or can you send me your smapi APK file?

**Comment by Rukkung1** [comment]
@Nhincaije How do I install it?

**Comment by Nhincaije** [comment]
> @Nhincaije How do I install it?

Yes, that's right, I want to know specifically how to install smapi for stardew valley v1.5.6.52

**Comment by WeirdDough0** [comment]
> > @Nhincaije How do I install it?
> 
> Yes, that's right, I want to know specifically how to install smapi for stardew valley v1.5.6.52

i also have SDV bought on Play store and i'm trying to open Smapi SDV laucher but every time i tried opening it, it always crashed. I'm using Redmi Note 10

**Comment by LWGITHUBID** [comment]
> > > @Nhincaije How do I install it?
> > 
> > 
> > Yes, that's right, I want to know specifically how to install smapi for stardew valley v1.5.6.52
> 
> i also have SDV bought on Play store and i'm trying to open Smapi SDV laucher but every time i tried opening it, it always crashed. I'm using Redmi Note 10

 This time. Smpinsdvn support .39,wait dev update.

**Comment by jsprdntn** [comment]
> > This version of SMAPI is not compatible with the latest version that I've installed from playstore.
> 
> If it is compatible, you just have to reinstall smapi and install it with "MTManager"
> 
> ![Screenshot_20231109-223027](https://private-user-images.githubusercontent.com/97956553/281926791-a72501bc-3821-4507-b500-83ae6444011d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI1MTYxNzksIm5iZiI6MTcwMjUxNTg3OSwicGF0aCI6Ii85Nzk1NjU1My8yODE5MjY3OTEtYTcyNTAxYmMtMzgyMS00NTA3LWI1MDAtODNhZTY0NDQwMTFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE0VDAxMDQzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJlMDJlMDAzYmJhMjc3MDNjZWExODA3MmRjYmQ5ZWI3MmMzZjc2OGZlYzQzZDA0YTNmM2NmMGY3NjY0YTZhZTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wBb4rdELl8hIcigcX0tRCdjZ7mbSoWfAAsvvufHMNyw)

Are you able to describe what you did to install SV and SMAPI with MT Manager? Do you mean you just used MTManager to install the apks? I did this and I'm getting the same errors for .39 or .52
